### ISSUE TITLE
int array throws exception

### ISSUE THREAD

**Issue by msegmx** [orhanobut/logger#113]
This code 

```
@Override
	public void onUpdate(Context context, AppWidgetManager widgetManager, int[] widgetIds) {
		Logger.d(widgetIds);

```
throws this exception :

```
Caused by: java.lang.ClassCastException: int[] cannot be cast to java.lang.Object[]
at com.orhanobut.logger.LoggerPrinter.d(LoggerPrinter.java:116)
at com.wolf359apps.redditastic2dev.R2AppWidgetProvider.onUpdate(R2AppWidgetProvider.java:24)
```

**Comment by orhanobut** [comment]
@msegmx Thanks for the bug report. I'll fix this one with the upcoming version.

**Comment by orhanobut** [comment]
Fixed with the new version (2.1.0)
